docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -56,9 +56,13 @@ DataImportCrons are widely used for golden images and likely serve as a source t
 
 ## Mitigation
 
-A common issue when opting in to golden images auto-polling is not having a default storage class set.  
+### No storage class defined (default, or per DataImportCron)
+
+A common issue when opting in to golden images auto-polling is not having a _default storage class_ set.
+
 Ensure you have a default storage class set in the cluster *or*  
-if you're using a custom DataImportCron, verify there is an explicit storage class set in the DataImportCron definition:
+If you're using a custom DataImportCron, and no default storage class is set,  verify there is an explicit storage class set in the DataImportCron definition:
```

Comment:
If you're using a custom DataImportCron, and no default storage class is set,  verify there is an explicit storage class set in the DataImportCron definition: -> If you're using a custom DataImportCron and no default storage class is set, verify that there is an explicit storage class set in the DataImportCron definition:

Reply:
fixed